nrovalidate/validator/kubeletconfig.go

```diff
@@ -20,20 +20,78 @@ import (
 	"context"
 	"fmt"
 
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog/v2"
 	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/k8stopologyawareschedwg/deployer/pkg/deployer/platform"
 	deployervalidator "github.com/k8stopologyawareschedwg/deployer/pkg/validator"
+	hypershiftconsts "github.com/openshift-kni/numaresources-operator/internal/hypershift/consts"
 	"github.com/openshift-kni/numaresources-operator/pkg/kubeletconfig"
+	"github.com/openshift-kni/numaresources-operator/test/utils/configuration"
```

Comment:
again, we must not depend on `test/...` outside the testing tree (if we did already before this PR, that's a huge bug we need to fix ASAP)